src/IBM.WatsonDeveloperCloud/Http/Extensions/UriExtensions.cs

```diff
@@ -30,7 +30,7 @@ public static Uri WithArguments(this Uri uri, params KeyValuePair<string, object
                 from argument in arguments
                 let key = WebUtility.UrlEncode(argument.Key)
                 let value = argument.Value != null ? WebUtility.UrlEncode(argument.Value.ToString()) : string.Empty
-                select key + "=" + value
+                select key + "=" + value.ToLower()

```

Comment:
@atilatosta This is making our Speech to Text integration tests fail for sessions. The query param for model is case sensitive. Is there another solution?